system/src/system_publish_vitals.cpp

```diff
@@ -0,0 +1,129 @@
+
+#include "system_publish_vitals.h"
+
+#include <limits>
+
+#include "logging.h"
+#include "spark_protocol_functions.h"
+#include "system_threading.h"
+
+using namespace particle::system;
+
+template <class Timer>
+const typename VitalsPublisher<Timer>::publish_fn_t VitalsPublisher<Timer>::_publishVitals =
+    std::bind(spark_protocol_post_description, spark_protocol_instance(),
+              particle::protocol::DESCRIBE_METRICS, nullptr);
```

Comment:
This can be a regular static function, there's no need to wrap `spark_protocol_post_description()` into an `std:function`

Reply:
good point

Reply:
good point, it has been removed